packages/rollup-plugin/src/index.js

```diff
@@ -41,10 +41,18 @@ export default function stylexPlugin({
   ...options
 }: PluginOptions = {}): Plugin<> {
   let stylexRules: { [string]: $ReadOnlyArray<Rule> } = {};
+  let isWatchMode = process.argv.some((c) => ['--watch', '-w'].includes(c));
   return {
     name: 'rollup-plugin-stylex',
+    options(option) {
+      if (option.watch && !isWatchMode) {
+        isWatchMode = true;
+      }
+    },
     buildStart() {
-      stylexRules = {};
+      if (!isWatchMode) {
+        stylexRules = {};
```

Comment:
The challenge with this approach is that it won't detect deleted files anymore.

Reply:
`shouldTransformCachedModule` should be generating the CSS again anyway. Need to debug why its not.

Reply:
Actually,`shouldTranformCacheModule` is a perf hook for rollup in `watch` mode, as far as i can know this hook does not affect the result. when i debug this case and try comment out `shouldTransformCacheModule` logic nothing be changed. In my mind, watch mode alway use in user development mode, so the dead code elimination is unnecessary. If we need it we can using result code import statments to rewrite all `stylexRules` in watch mode.

Reply:
oh i know. i find another way to resolve this.

Reply:
I removed the logic for clearing the cache and instead used `this.load` to re-insert the stylesheet. I think this is correct and consistent with the behavior of build.